File: Kachelmann/DayForecast.cs
using System;

namespace RaspPiTest.Kachelmann
{
    public class DayForecast
    {
        public DateTime Date { get; set; }
        public Sunshine Morning { get; set; }
        public Sunshine Afternoon { get; set; }
        public Sunshine Night { get; set; }
        public float MaxTemperature { get; set; }
        public float MinTemperature { get; set; }
        public Wind Wind { get; set; }
        public float ExpectedRain { get; set; }
    }
}

File: App/pidrei-ng-app/src/shared/classes/day-forecast.ts
import { WindInformation } from './wind-information';

export class DayForecast {
    public date: Date;
    public morning: number;
    public afternoon: number;
    public night: number;
    public maxTemperature: number;
    public minTemperature: number;
    public wind: WindInformation;
    public expectedRain: number;
}
